Signal/src/ViewControllers/AttachmentApprovalViewController.swift

```diff
@@ -140,10 +147,11 @@ class AttachmentApprovalViewController: UIViewController, OWSAudioAttachmentPlay
         audioPlayButton.autoSetDimension(.height, toSize:buttonSize)
         subviews.append(audioPlayButton)
 
-        if let fileExtensionLabel = createFileExtensionLabel() {
+        if let fileNameLabel = createFileNameLabel() {
+            subviews.append(fileNameLabel)
+        } else if let fileExtensionLabel = createFileExtensionLabel() {
             subviews.append(fileExtensionLabel)
         }
```

Comment:
I think this is okay. If an attachment has neither a file name nor a recognizable file type, I think it's fine just to show the file size.